components/RepoList.js

```diff
@@ -0,0 +1,77 @@
+import 'expo';
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  StyleSheet,
+  Text,
+  View,
+  ScrollView,
+  Platform,
+  TouchableOpacity,
+} from 'react-native';
+import {
+  Actions,
+} from 'react-native-router-flux';
+import Repo from './Repo';
+
+class RepoList extends React.Component {
+  state = {
```

Comment:
Since we're managing the repos' list data with redux, you don't need this state anymore.